### PR TITLE
de-DE: Add gr33ndev as maintainer

### PR DESCRIPTION
**Pull request by Gr33ndev**
Add myself as de-DE maintainer, so I get informed when there is something to translate

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/changed_field_template.md [hunk 1]
```diff
@@ -38,4 +38,4 @@ Updated in:
 - [ ] zh-TW
 
 Notification when new strings arrive:
-@Omaranwa @J0anJosep @LPSGizmo @Wuzzy2 @tellovishous @dimateos @groenroos @rmnvgr @LucaRed @telk5093 @Goddesen @Gymnasiast @marcinkunert @Tupaschoal @LPR-Michael @daihakken
+@Omaranwa @J0anJosep @LPSGizmo @Wuzzy2 @tellovishous @dimateos @groenroos @rmnvgr @LucaRed @telk5093 @Goddesen @Gymnasiast @marcinkunert @Tupaschoal @LPR-Michael @daihakken @gr33ndev
```
Comment (tupaschoal) [inline .github/ISSUE_TEMPLATE/changed_field_template.md:41]:
```suggestion
@Omaranwa @J0anJosep @LPSGizmo @Wuzzy2 @gr33ndev @tellovishous @dimateos @groenroos @rmnvgr @LucaRed @telk5093 @Goddesen @Gymnasiast @marcinkunert @Tupaschoal @LPR-Michael @daihakken
```

File: .github/ISSUE_TEMPLATE/new_and_changed_field_template.md [hunk 2]
```diff
@@ -44,4 +44,4 @@ Updated in:
 - [ ] zh-TW
 
 Notification when new strings arrive:
-@Omaranwa @J0anJosep @LPSGizmo @Wuzzy2 @tellovishous @dimateos @groenroos @rmnvgr @LucaRed @telk5093 @Goddesen @Gymnasiast @marcinkunert @Tupaschoal @LPR-Michael @daihakken
+@Omaranwa @J0anJosep @LPSGizmo @Wuzzy2 @tellovishous @dimateos @groenroos @rmnvgr @LucaRed @telk5093 @Goddesen @Gymnasiast @marcinkunert @Tupaschoal @LPR-Michael @daihakken @gr33ndev
```
Comment (tupaschoal) [inline .github/ISSUE_TEMPLATE/new_and_changed_field_template.md:47]:
```suggestion
@Omaranwa @J0anJosep @LPSGizmo @Wuzzy2 @gr33ndev @tellovishous @dimateos @groenroos @rmnvgr @LucaRed @telk5093 @Goddesen @Gymnasiast @marcinkunert @Tupaschoal @LPR-Michael @daihakken
```

File: .github/ISSUE_TEMPLATE/new_field_template.md [hunk 3]
```diff
@@ -38,4 +38,4 @@ Updated in:
 - [ ] zh-TW
 
 Notification when new strings arrive:
-@Omaranwa @J0anJosep @LPSGizmo @Wuzzy2 @tellovishous @dimateos @groenroos @rmnvgr @LucaRed @telk5093 @Goddesen @Gymnasiast @marcinkunert @Tupaschoal @LPR-Michael @daihakken
+@Omaranwa @J0anJosep @LPSGizmo @Wuzzy2 @tellovishous @dimateos @groenroos @rmnvgr @LucaRed @telk5093 @Goddesen @Gymnasiast @marcinkunert @Tupaschoal @LPR-Michael @daihakken @gr33ndev
```
Comment (tupaschoal) [inline .github/ISSUE_TEMPLATE/new_field_template.md:41]:
```suggestion
@Omaranwa @J0anJosep @LPSGizmo @Wuzzy2 @gr33ndev @tellovishous @dimateos @groenroos @rmnvgr @LucaRed @telk5093 @Goddesen @Gymnasiast @marcinkunert @Tupaschoal @LPR-Michael @daihakken
```

**Review by tupaschoal — COMMENTED** [review]
I keep the notification list in the same order as the languages on the readme, just for ease of cleanup when needed.

**Comment by Wuzzy2** [comment]
Welcome to the translation team!

**Pull request merged**